spec/definition-export-spec.js

```diff
@@ -0,0 +1,32 @@
+const sources = ["bikeshed", "respec"];
+
+const docTypes = new Map();
+beforeAll(async () => {
+  for (let source of sources) {
+    docTypes.set(source, createIframe(source));
+  }
+});
+
+afterAll(() => {
+  cleanUp(docTypes.values());
+});
+
+for (const type of sources) {
+  describe(`${type} - simple definition`, () => {
```

Comment:
could we have the expected extracted definitions loaded from separate JSON files instead? that way extracting tools could re-use the test suite to make sure they can generate the said extracts.

(speaking of which, reffy has a [related test suite](https://github.com/w3c/reffy/blob/master/tests/extract-dfns.js) that may serve as inspiration, and ideally be at least partially taken over by this one)

Reply:
Yes, that's a great idea. I'd probably want to write out a few more cases to see what kind of structure we need and then convert them to a JSON file.